<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpBoxSummary">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>441</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>요약</string>
    </property>
    <widget class="QLabel" name="lblLabelTotalAssets">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>21</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>총 자산</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblLabelBuy">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>44</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>총 매입</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblLabelEvaluation">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>66</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>총 평가</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblLabelEvalProfit">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>89</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>평가 손익</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblLabelEvalProfitRatio">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>111</y>
       <width>64</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>평가 수익율</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblLabelRealProfit">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>134</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>실현 손익</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRealProfit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblEvalProfit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>85</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalAssets">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>17</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalBuy">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblEvalProfitRatio">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>107</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalEvaluation">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>62</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>20</width>
       <height>131</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAccountInfo">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBoxCtrlPanel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>컨트롤 패널</string>
    </property>
    <widget class="QLabel" name="lblLabelRunningTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Running Time: </string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblRunningTime">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0:00:00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnd">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>종료</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpBoxSellPriceControl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>매도가 설정</string>
     </property>
     <widget class="QLabel" name="lblUpperLimit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>매도상한(%):</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLowerLimit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>매도 하한(%):</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSellPriceCancel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSellPriceApply">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>적용</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUpperLimit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLowerLimit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tblWgtTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>841</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>137</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>34</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>종목 별</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>보유량</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>평균 매입가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>현재가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>평가 손익</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>수익률</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionLogIn"/>
    <addaction name="actionLogOut"/>
    <addaction name="actionLogState"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogIn">
   <property name="text">
    <string>로그인</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="text">
    <string>로그아웃</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
  <action name="actionLogState">
   <property name="text">
    <string>로그인 상태</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
